packages/components/organisms/f-form/src/components/_tests/Form.test.js

```diff
@@ -43,21 +18,66 @@ describe('Form', () => {
         expect(wrapper.exists()).toBe(true);
     });
 
+
+    it('should have one form with method "post"', () => {
+        // Arrange
+        const wrapper = shallowMount(FForm, { propsData });
+
+        // Act
+        const forms = wrapper.findAll('form');
+
+        // Assert
+        expect(forms.length).toBe(1);
```

Comment:
It it likely that there will be two or more forms? Not sure it's worth checking for that